docs/dabp/release-notes/index.md

```diff
@@ -0,0 +1,18 @@
+# Release notes
+
+This section contains release notes for Onegini DABP.
+
+In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [Onegini Support](https://support.onegini.com).
+
+## Release date: 2021-04-14
+* After removing a user from a group, if he is no longer of any group his account will be removed from CIM
+* Translation fixes and improvements
+* Minor UI EventHandler
```

Comment:
Eventhandler?

Reply:
Why did you change it from improvements?

Reply:
Stupid intellij, sometimes when I try to paste stuff it puts things in random places because the focus does not switch correcly when switching multiple windows... fixed.